test/unit/crdReplicator/crdReplicator-operator_test.go

```diff
@@ -33,17 +34,28 @@ var (
 func setupDispatcherOperator() error {
 	var err error
 	localDynClient := dynamic.NewForConfigOrDie(k8sManagerLocal.GetConfig())
+	localDynFac := dynamicinformer.NewFilteredDynamicSharedInformerFactory(localDynClient, crdReplicator.ResyncPeriod, metav1.NamespaceAll, func(options *metav1.ListOptions) {
+		//we want to watch only the resources that should be replicated on a remote cluster
+		klog.Infof("printing labels: %s", options.LabelSelector)
+		if options.LabelSelector == "" {
+			options.LabelSelector = crdReplicator.LocalLabelSelector + "=true"
+		} else {
+			options.LabelSelector = options.LabelSelector + "," + crdReplicator.LocalLabelSelector + "=" + "true"
```

Comment:
In order to make the string building more efficient, we could use either `strings.Builder` or `strings.Join()`

Reply:
thanks for the suggestion. Using `strings.Join()` instead of the `+` operator.